app/main/helpers/suppliers.py

```diff
@@ -121,3 +122,15 @@ def format_g12_recovery_time_remaining(time_to_deadline: timedelta) -> str:
         unit += 's'
 
     return f'{number} {unit}'
+
+
+def count_unsubmitted_g12_recovery_drafts(data_api_client: DataAPIClient, supplier_id: int) -> int:
+    unsubmitted_recovery_drafts = [
+        draft for draft in
+        data_api_client.find_draft_services_iter(
```

Comment:
Can we reduce the number of network requests needed by using `find_draft_services_by_framework_iter`, which can get the API to filter by status for us?

Reply:
That sounds very sensible